WebRoot/js/zimbraAdmin/accounts/controller/ZaAccountListController.js

```diff
@@ -975,14 +975,19 @@ function (item) {
 	//check the passwords, if they are ok then save the password, else show error
 	if(this._chngPwdDlg) {
 		try {
-			if(!this._chngPwdDlg.getPassword() || this._chngPwdDlg.getPassword().length < 1) {
-				ZaApp.getInstance().dialogs["errorMsgDlg"] = new ZaMsgDialog(ZaApp.getInstance().getAppCtxt().getShell(), null, [DwtDialog.OK_BUTTON],null,ZaId.CTR_PREFIX + ZaId.VIEW_ACCTLIST + "_errorMsg");							
+			const usrName = item.name.split('@')[0];
```

Comment:
1. do not use `const`. It has been introduced since ES6/ECMAScript 2015.
2. rename the variable to userName if it means user name and it does not conflict.
```suggestion
			var userName = item.name.split('@')[0];
```

Reply:
Changes Done

Reply:
Let me know the reason why the variable name is not `userName` but `usrName`.

Reply:
Let me know the reason why the variable name is not `userName` but `usrName`.

Reply:
we can have the variable name as userName.Made changes accordingly.